spec/taxonomy_sidebar_spec.rb

```diff
@@ -123,16 +123,48 @@
           ]
         )
       end
+    end
+
+    context 'given there are repeated related links across different parent taxons' do
+      before do
+        # First taxon should have links A and B
+        stub_request(
+          :get,
+          %r{.*search.json?.*\&filter_taxons%5B%5D=taxon-a.*}
+        ).to_return(
+          status: 200,
```

Comment:
(I don't think you need to provide the `status` or `header` arguments? They're just visual noise here. Same below.)